events/codepipeline_cloudwatch.go

```diff
@@ -42,8 +42,8 @@ const (
 )
 
 // CodePipelineEvent is documented at:
-// https://docs.aws.amazon.com/AmazonCloudWatch/latest/events/EventTypes.html#codepipeline_event_type
-type CodePipelineCloudWatchEvent struct {
+// https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-service-event.html#codepipeline_event_type
```

Comment:
The content is different than what it was when it was added in the first place. May as well remove the link if there's not an equivalent anymore